includes/class-wporg-notifications.php

```diff
@@ -94,28 +95,53 @@ function ( $message, $original_id ) {
 				10,
 				2
 			);
+			add_filter(
+				'wporg_load_mentions_list',
+				function( $result, $comments, $locale, $original_id ) {
+					$validator_email_addresses  = WPorg_GlotPress_Notifications::get_validator_email_addresses_for_original_id( $locale, $original_id );
+					$commenters_email_addresses = array_values( GP_Notifications::get_commenters_email_addresses( $comments, $validator_email_addresses ) );
+
+					$all_email_addresses = array_merge(
+						$validator_email_addresses,
+						$commenters_email_addresses
+					);
+
+							$users = array_map(
+								function( $email ) {
+									$user = get_user_by( 'email', $email );
+									return array(
+										'ID'            => $user->ID,
+										'user_login'    => $user->user_login,
+										'user_nicename' => $user->user_nicename,
+										'display_name'  => '',
+										'source'        => array( 'translators' ),
```

Comment:
Would it be possible to vary this depending on whether they are validators? Ideally also display that in the mention but probably outside of scope for this PR.
![Screenshot 2022-09-27 at 11 42 19](https://user-images.githubusercontent.com/203408/192493305-06872c24-5101-483f-b6fa-bd15ab8c2ab4.png)

Reply:
Yea, this makes sense, we can have this in another PR